cfgov/v1/models/home_page.py

```diff
@@ -54,4 +56,23 @@ def get_category_name(self, category_icon_name):
 
     def get_context(self, request):
         context = super(HomePage, self).get_context(request)
+        context['latest_updates'] = self.get_latest_updates()
```

Comment:
You might want to pass the request in here so that you can also [filter by the current site](https://docs.wagtail.io/en/v1.13.4/reference/pages/queryset_reference.html#wagtail.wagtailcore.query.PageQuerySet.in_site), so as not to include pages in the Trash.